File: GSI QA testing tool/UI/UI_010_IndividualPage.cs
using OpenQA.Selenium;
using System;
using System.Collections.Generic;
using System.Diagnostics;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace GSI_QA_testing_tool.UI
{
    internal class UI_010_IndividualPage
    {
        public static void GoTo(IWebDriver driver)
        {

            Finder.ClickIt(driver, "//a[@id='ctl00_hlNotRegistered']");
            Finder.ClickIt(driver, "//button[@id='btnguestlogina']");
            Finder.ClickIt(driver, "//input[@id='btnIndRegistration']");


            //if (driver.FindElements(By.XPath("//a[@id='ctl00_hlNotRegistered']")).Count != 0)
            //{
            //    driver.FindElement(By.XPath("//a[@id='ctl00_hlNotRegistered']")).Click();
            //}
            //else if (driver.FindElements(By.XPath("//button[@id='btnguestlogina']")).Count != 0)
            //{
            //    driver.FindElement(By.XPath("//button[@id='btnguestlogina']")).Click();
            //}
            //else
            //{
            //    driver.FindElement(By.XPath("//input[@id='btnIndRegistration']")).Click();
            //}
        }
    }
}

File: GSI QA testing tool/UI/UI_010_IndividualPage.vb
Imports OpenQA.Selenium
Imports System
Imports System.Collections.Generic
Imports System.Diagnostics
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks

Namespace GSI_QA_testing_tool.UI
    Friend Class UI_010_IndividualPage
        Public Shared Sub [GoTo](ByVal driver As IWebDriver, ByVal js As IJavaScriptExecutor)
            If Data._StopAt.Contains("Individual Page") Then
                Data.StopAtDialog()
            End If

            If Data._Staff = True Then
                Finder.SendText(driver, "//input[@id='txtUserName']", Data._StaffLogin)
                Finder.SendText(driver, "//input[@id='txtPassword']", Data._StaffPass)
                Finder.JSClickIt(driver, "#ctl00_Main_content_cmdLoginButt", js)
            Else
                Finder.JSClickIt(driver, "#ctl00_hlNotRegistered", js)
                Finder.JSClickIt(driver, "#btnguestlogina", js)
                Finder.JSClickIt(driver, "#btnIndRegistration", js)
            End If
        End Sub
    End Class
End Namespace
